a10_octavia/controller/housekeeping/house_keeping.py

```diff
@@ -27,6 +29,9 @@
 
 LOG = logging.getLogger(__name__)
 CONF = cfg.CONF
+mp_mgr = multiprocessing.Manager()
+hk_ctx_map = mp_mgr.dict()
```

Comment:
Please help test 2 more unit-test cases which related to multi-process synchronization.

- In hardware flow
1. hang the PendingCleanUp flow (by rpdb or sleep…etc)
2. create a new loadbalancer
3. The new loadbalancer creation **will not be locked for waiting clean up process**

- In vthunder flow
1. hang the PendingCleanUp flow (by rpdb or sleep…etc)
2. create a new loadbalancer
3. The new loadbalancer creation **will be locked for waiting clean up process**

---

Thing should be work fine if multiprocessing.Manager works as expected. But if not we either:
- find another solution, or
- if it take too long to find another solution. We **just mark this feature only supported for hardware flow in design document**.

Reply:
Updated the code to don't cleanup vThunder flow resources.
Also added the limitation in design document